src/app/lists/ListDataview.tsx

```diff
@@ -19,49 +14,81 @@ import TooltipCell from "@/app/lists/TooltipCell";
 import TableSurface from "@/components/Tables/TableSurface";
 import CommentBox from "@/app/lists/CommentBox";
 
-export type ListRow = Schema["lists"] & Datum;
+interface ListRow {
+    itemName: string;
```

Comment:
Should this be `Schema["lists"]["item_name"]`?

Reply:
I would probably want a type error here if it was changed to something that wasn't a string.